### PR TITLE
tests: Use custom kernel for all tests

### PR DESCRIPTION
**Pull request by rbradford**
This should reduce the integration testing time considerably. When a
custom kernel is no longer required we can pull kernel from tarball
again.

Fixes: #100

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by rbradford** [comment]
Seems we get less entropy with the different kernel. I've checked the .config fles and there is no obvious difference so I think we might be looking at a change in behaviour across the versions of the kernels themselves.

**Comment by rbradford** [comment]
Interestingly on a smaller VM D2v3 vs D32sv3 those tests do not fail.

**Comment by rbradford** [comment]
Shaves another 2 minutes off our build time!

File: src/main.rs [hunk 1]
```diff
@@ -424,7 +424,7 @@ mod tests {
             aver_eq!(tb, guest.get_cpu_count(), 1);
             aver_eq!(tb, guest.get_initial_apicid(), 0);
             aver!(tb, guest.get_total_memory() > 496_000);
-            aver!(tb, guest.get_entropy() >= 1000);
+            aver!(tb, guest.get_entropy() >= 900);
```
Comment (sboeuf) [inline src/main.rs]: Well there are two ways to solve this, either we lower this value or we wait a bit more, right?
I mean we have a virtio-rng device supposed to seed the entropy, right?

Comment (rbradford) [reply src/main.rs]: We're getting the lower number 'cause we're booting quicker. Which is a good thing! There is no meaning in the number (other that very low being very bad.) The original 1000 is just something that worked before.

Comment (sboeuf) [reply src/main.rs]: Sounds good!

**Comment by sboeuf** [comment]
@sameo PTAL

**Pull request merged**